xml/ns-System.Text.Json.xml

```diff
@@ -1,6 +1,6 @@
 <Namespace Name="System.Text.Json">
   <Docs>
-    <summary>To be added.</summary>
+    <summary>The System.Text.Json namespace provides high-performance, low-allocating, and RFC-compliant capabilities to process JavaScript Object Notation (JSON) which includes serializing objects to JSON text and deserialize JSON text to objects, with UTF-8 support built-in. It also provides types to read and write JSON text encoded as UTF-8, and to create an in-memory document object model (DOM), that is read-only, for random access of the JSON elements within a structured view of the data.</summary>
```

Comment:
```suggestion
    <summary>The System.Text.Json namespace provides high-performance, low-allocating, and RFC-compliant capabilities to process JavaScript Object Notation (JSON), which includes serializing objects to JSON text and deserializing JSON text to objects, with UTF-8 support built-in. It also provides types to read and write JSON text encoded as UTF-8, and to create a read-only in-memory document object model (DOM) for random access of the JSON elements within a structured view of the data.</summary>
```

Reply:
Given we are considering adding modifiable Json DOM APIs in vNext, maybe we can drop the "readonly" specifier. Avoids having the problem of the namespace description being too (current implementation) specific and requiring updating when more things get added.